returnn/tf/layers/basic.py

```diff
@@ -842,8 +842,9 @@ def get_out_data_from_opts(
 
 class SliceNdLayer(_ConcatInputLayer):
   """
-  This takes out a slice-range from some axis,
-  e.g. ``x[start:start + size]``.
```

Comment:
Leave the old comment in. I think this is helpful to understand what a slice actual is.

Always start with the simple and common examples first, and then go over to the more generic examples.